src/pycram/process_modules/icub_process_modules.py

```diff
@@ -0,0 +1,687 @@
+import math
+from threading import Lock
+from typing import Union
+from typing_extensions import Any
+import actionlib
+from .default_process_modules import DefaultMoveJoints, DefaultMoveArmJoints, DefaultMoveTCP, DefaultNavigation, \
+    DefaultMoveHead, DefaultDetecting, DefaultMoveGripper
+from .. import world_reasoning as btr
+import numpy as np
+from ..process_module import ProcessModule, ProcessModuleManager
+from ..external_interfaces.ik import request_ik
+from ..ros.logging import logdebug
+from ..utils import _apply_ik
+from ..local_transformer import LocalTransformer
+from ..designators.object_designator import ObjectDesignatorDescription
+from ..designators.motion_designator import MoveMotion, LookingMotion, \
+    DetectingMotion, MoveTCPMotion, MoveArmJointsMotion, WorldStateDetectingMotion, MoveJointsMotion, \
+    MoveGripperMotion, OpeningMotion, ClosingMotion
+from ..robot_description import RobotDescription
+from ..datastructures.world import World
+from ..world_concepts.world_object import Object
+from ..datastructures.pose import Pose
+from ..datastructures.enums import JointType, ObjectType, Arms, ExecutionType, GripperState
+from ..external_interfaces import giskard
+from ..external_interfaces.robokudo import *
+import yarp
+
+from pycram.yarp_utils.yarp_networking import NO_ACK_VOCAB, ACK_VOCAB, open_rpc_client_port, open_buffered_bottle_port, \
+    init_yarp_network
+
+def update_hand(hand_values,ctp_port):
+    yarp_bottle_msg: yarp.Bottle = yarp.Bottle()
+    yarp_bottle_reply: yarp.Bottle = yarp.Bottle()
+    yarp_bottle_msg.addVocab32('c', 't', 'p', 'q')
+    yarp_bottle_msg.addVocab32('t', 'i', 'm', 'e')
+    yarp_bottle_msg.addFloat32(1.5)
+    yarp_bottle_msg.addVocab32('o', 'f', 'f')
+    yarp_bottle_msg.addInt32(7)
+    yarp_bottle_msg.addVocab32('p', 'o', 's')
+    target_loc: yarp.Bottle = yarp_bottle_msg.addList()
+    for i in hand_values:
+        target_loc.addFloat32(i)
+
+    print("command Ready to send to iCub  part tcp")
```

Comment:
Use pycram.ros.logging instead of print statements, throughout the whole file

Reply:
Done